tests/check_files/test_check_files/test_system_check_files.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import json
+import os
+import pytest
+from datetime import datetime
+from deepdiff import DeepDiff
+
+from wazuh_testing.tools.file import validate_json_file, read_json_file, write_json_file
+
+OUTPUT_FILE = f"system_checkfiles_{datetime.now().timestamp()}.json"
+
+
+@pytest.fixture
+def get_first_file(request):
+    """Allow to use the --before-file parameter in order to pass the file before making any changes to the environment.
+
+    Args:
+        request (fixture): Provide information on the executing test function.
+    """
+    return request.config.getoption('--before-file')
+
+
+@pytest.fixture
+def get_second_file(request):
+    """Allow to use the --after-file parameter in order to pass the file before making any changes to the environment.
+
+    Args:
+        request (fixture): Provide information on the executing test function.
+    """
+    return request.config.getoption('--after-file')
+
+
+@pytest.fixture
+def get_output_path(request):
+    """Allow to use the --output-path parameter to store the test result in the specified file.
+
+    Args:
+        request (fixture): Provide information on the executing test function.
+    """
+    return request.config.getoption('--output-path')
+
+
+def validate_and_read_json(file_path):
+    """Validate the JSON file passed as argument and return its content.
+
+    Args:
+        file_path (str): JSON file path.
+    """
+    if validate_json_file(file_path):
+        file_data = read_json_file(file_path)
+    else:
+        assert False, f"The file {file_path} is not a valid JSON."
```

Comment:
```suggestion
        Raise ValueError(f"The file {file_path} is not a valid JSON)
```

Reply:
Add Raises documentation too

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/2246/commits/d454cc3a22a879a834acba0ab4d9c663c5d2c84a